portfolio/src/main/webapp/index.html

```diff
@@ -53,12 +53,34 @@ <h2>Flowerboy</h2>
     <div>
       <h2>Ballers</h2>
       <p> 
-        The following is a collection of players who have inspired my love for the game. Soccer is 
-        the most popular sport in the world. This popularity means they are many boys like me who 
-        grow up loving the game and wishing to one day ascend to 
-        Evidently, its stars are extraordinary talents who have
-        managed to captivate billions of fans across the world. 
-      </p> 
+        The following is a collection of players amd teams who have inspired my love for the game. 
+        Soccer is the most popular sport in the world. This popularity means they are many boys 
+        like me who grow up loving the game and wishing to one day ascend to the heights where 
+        their skills can be admired alomgside the greatest players of their generation. Football 
+        stars receive contracts from major brands and can be seen in many adverts that are not 
+        related to soccer. Their prowess on the fields have allowed them to captivate billions of 
+        fans across the world. Evidently, the World Cup is the most-watched overall live event 
+        on television worldwide. Soccer brings nations together. The sport is a common language
+        understood universally. The joys and heartache felt are a testiment of people's love for
+        the beautiful game. 
+      </p>
+      <div class="row">
+        <div class="column">
+          <img src="images/brazil.jpg" width="300" height="300">
```

Comment:
Looks like you can have a css class to replace "width" and "height" so that all these duplicates will be removed.